components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/bean/ThrottleDataOfAPIAndApplicationValue.java

```diff
@@ -18,130 +18,60 @@
 */
 package org.wso2.carbon.apimgt.usage.client.bean;
 
+import java.util.Calendar;
+import java.util.List;
+
 public class ThrottleDataOfAPIAndApplicationValue {
-    private String api;
-    private String api_version;
-    private String context;
-    private String apiPublisher;
-    private String applicationName;
-    private String tenantDomain;
+    private int success_request_count;
+    private int throttle_out_count;
+    private List<String> api_year_month_week_day_facet;
```

Comment:
use this variable since it should be same to response coming from REST api when is parse back to the java.